src/universal/modules/summary/components/MeetingSummary/MeetingSummary.js

```diff
@@ -0,0 +1,89 @@
+import React, {PropTypes} from 'react';
+import withStyles from 'universal/styles/withStyles';
+import {css} from 'aphrodite-local-styles/no-important';
+import appTheme from 'universal/styles/theme/appTheme';
+
+import Type from 'universal/components/Type/Type';
+import MeetingMain from 'universal/modules/meeting/components/MeetingMain/MeetingMain';
+import MeetingSection from 'universal/modules/meeting/components/MeetingSection/MeetingSection';
+import MeetingPhaseHeading from 'universal/modules/meeting/components/MeetingPhaseHeading/MeetingPhaseHeading';
+import SummaryEmailPreview from 'universal/modules/meeting/components/SummaryEmailPreview/SummaryEmailPreview';
+import SummaryFirstTime from 'universal/modules/meeting/components/SummaryFirstTime/SummaryFirstTime';
+import SummaryQuickStats from 'universal/modules/meeting/components/SummaryQuickStats/SummaryQuickStats';
+import {makeSuccessExpression} from 'universal/utils/makeSuccessCopy';
+
+const MeetingSummary = (props) => {
```

Comment:
I had one team leader and one participant. My team leader had the team summary rendered, but the participant was redirected to the team lobby.

Can we implement it so everybody sees the team summary and has a link to return to their dashboard?